dlhub_sdk/client.py

```diff
@@ -474,3 +474,5 @@ def clear_funcx_cache(self, servable=None):
             self.fx_cache = {}
 
         return self.fx_cache
+
```

Comment:
Extra lines at the end should through a Flake8 error, which I'd pay attention to then fix.